examples/js/controls/OrbitControls.js

```diff
@@ -17,7 +17,10 @@ THREE.OrbitControls = function ( object, domElement ) {
 
 	this.object = object;
 
-	this.domElement = ( domElement !== undefined ) ? domElement : document;
+	if ( domElement === undefined ) console.error( 'THREE.OrbitControls: The second parameter "domElement" is now mandatory.' );
+	if ( domElement === document ) console.warn( 'THREE.OrbitControls: "document" cannot be used as the target "domElement". Please use "renderer.domElement" instead.' );
```

Comment:
Then checks for "document" elsewhere in the file need to be removed, right?

And this line should be a `console.error`, I guess.

Reply:
Good point 👍